setup.py

```diff
@@ -46,8 +46,8 @@ def readme():
       cmdclass=versioneer.get_cmdclass(),
       use_2to3=False,
 
-      maintainer='Jens H Nielsen',
-      maintainer_email='Jens.Nielsen@microsoft.com',
+      maintainer='QCoDeS Core Developers',
```

Comment:
@jenshnielsen I'm happy to leave you has the maintainer if you think an actual name here is more appropriate.

Reply:
No I think this is good.